src/country.rs

```diff
@@ -234,6 +236,7 @@ impl Country {
             Guatemala => f!(c(4), c(20)),
             Hungary => f!(n(24)),
             Iceland => f!(n(22)),
+            Internet => f!(n(4), c(8)),
```

Comment:
The bank code of an Internet IBAN is also alpha numeric
```suggestion
            Internet => f!(c(12)),
```